internal/noderesourcetopology/attributes.go

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package noderesourcetopology
+
+const (
+	TopologyManagerPolicyAttribute = "topologyPolicyAttribute"
+	TopologyManagerScopeAttribute  = "topologyScopeAttribute"
+)
+
+//   - `restricted`: kubelet only allows pods with optimal NUMA node alignment for
+//     requested resources;
+//   - `best-effort`: kubelet will favor pods with NUMA alignment of CPU and device
+//     resources;
+//   - `none`: kubelet has no knowledge of NUMA alignment of a pod's CPU and device resources.
+//   - `single-numa-node`: kubelet only allows pods with a single NUMA alignment
+//     of CPU and device resources.
```

Comment:
where is the source for these comments?

Reply:
From kubelet config package https://github.com/kubernetes/kubelet/blob/6f5021b1251c0e562b5753a402bab61ec4b07596/config/v1beta1/types.go#L374

Reply:
OK, this content is unlikely to change anytime soon. I'd backlink the original source here (and below for scopes) but not worth resubmitting for this little change.